tests/test_filtsmooth/test_statespace/test_continuous/test_continuousmodel.py

```diff
@@ -1,9 +1,10 @@
 import unittest
 
 import numpy as np
-import probnum.random_variables as rvs
 
+import probnum.random_variables as rvs
 from probnum.filtsmooth.statespace.continuous import continuousmodel
+from probnum.random_variables import Dirac, Normal
```

Comment:
This line seems to be responsible for the pylint check to fail, probably due to the merge of the updated master branch. After removing it the CI should be satisfied and we should be able to merge the PR :)